app/questionnaire/rules.py

```diff
@@ -256,7 +256,10 @@ def _get_when_rule_value(
     elif "meta" in when_rule:
         value = get_metadata_value(metadata, when_rule["meta"])
     elif "id_selector" in when_rule:
-        value = getattr(list_store.get(when_rule["list"]), when_rule["id_selector"])
+        if when_rule["id_selector"] == "first":
+            value = list_store.get_first_item_from_list(when_rule["list"])
+        else:
+            value = getattr(list_store.get(when_rule["list"]), when_rule["id_selector"])
```

Comment:
This could be reverted if the `first` when rule id_selector was used to access a `first` attribute on the ListModel instance

Reply:
So on `ListModel`, add:

```Python
@property
def first(self):
    return self.items[0]
```